asset-template.json.hb

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Block Coding Demos",
+  "description": "These are demos for the Godot Block Coding Plugin. Intended to show the potential and find limitations on the plugin, in order to instruct its development.",
+  "category_id": "10",
+  "godot_version": "4.3",
```

Comment:
Yep, this has to be 4.3 because one of the demos is using the new Parallax2D node.